DapperDoodle/QueryBuilders.cs

```diff
@@ -0,0 +1,68 @@
+﻿using System;
+using System.Data;
+using System.Text;
+using DapperDoodle.Exceptions;
+
+namespace DapperDoodle
+{
+    public static class QueryBuilders
+    {
+        public static string BuildSelectStatement<T>(this Query query)
+        {
+            return BuildSelectStatement<T>(query, null, Case.Lowercase);
+        }
+        
+        public static string BuildSelectStatement<T>(this Query query, string table, string clause)
+        {
+            return BuildSelectStatement<T>(query, table, Case.Lowercase, clause);
+        }
+        
+        public static string BuildSelectStatement<T>(this Query query, Case @case)
```

Comment:
Curious why the Case variable is prefixed with an @ symbol, do you mind explaining? I've seen it in other places as well. Is the word "case" a C# type or reserved word?

Reply:
@Krummelz Yes indeed, "case" is a reserved keyword for use within switch-cases. But you do raise a good point that my naming needs to be more consistent because I see that in some places I used "@case" and in others I used "casing". So I will stick to one and use that.